lib/config.ts

```diff
@@ -153,6 +165,16 @@ class Config {
                 header: res._headers
             };
         };
+        // Add client cert serializer
+        bunyan.stdSerializers['cert'] = function(cert) {
+            if (!cert) {
```

Comment:
This block was a bit confusing to read.  I would either have the second return in an `} else {` block or do the following:
```typescript
return cert && {
    CN: cert.subject.CN,
    fingerprint: cert.fingerprint
}
```